shinken/objects/host.py

```diff
@@ -874,6 +881,31 @@ def add_host_chk_dependency(self, h, status, timeperiod, inherits_parent):
     def add_service_link(self, service):
         self.services.append(service)
 
+
+    def __repr__(self):
+        return '<Host host_name=%r name=%r use=%r />' % (
+            getattr(self, 'host_name', None),
+            getattr(self, 'name', None),
+            getattr(self, 'use', None))
+
+    __str__ = __repr__
+
+    def is_excluded_for(self, svc):
```

Comment:
Do you think you need a whole service object? You only use the service description and the is_tpl() (see my comment below). It would avoir the fake_service you are doing afterwards.